e2etest/GuestProxyAgentTest/LinuxScripts/GuestProxyAgentExtensionValidation.sh

```diff
@@ -0,0 +1,116 @@
+        done 
+        break
+    fi
+    ((elapsed += interval))
+    if [[ $elapsed -ge $timeout ]]; then
+        echo "Timeout reached. Exiting the loop."
+        break
+    fi
+    sleep 5
+done 
+extensionVersion=$(echo "$PIRExtensionFolderPath" | grep -oP '(\d+\.\d+\.\d+)$')
+echo "extensionVersion=$extensionVersion"
+guestProxyAgentExtensionVersion=true
+
+echo "detecting os and installing jq" #TODO: needs to be revisited if we support other distros
+os=$(hostnamectl | grep "Operating System")
+echo "os=$os"
+if [[ $os == *"Ubuntu"* ]]; then
+    for  i in {1..3}; do
+        echo "start installing jq via apt-get $i"
+        sudo apt update
+        sudo apt-get install -y jq
+        sleep 10
+        install=$(apt list --installed jq)
+        echo "install=$install"
+        if [[ $install == *"jq"* ]]; then
+            echo "jq installed successfully"
+            break
+        fi
+    done
+else
+    for  i in {1..3}; do
+        echo "start installing jq via yum $i"
+        sudo yum -y install jq
+        sleep 10
+        install=$(yum list --installed jq)
+        echo "install=$install"
+        if [[ $install == *"jq"* ]]; then
+            echo "jq installed successfully"
+            break
+        fi
+    done
+fi
+
+echo "TEST: Check that status file is success with 5 minute timeout"
+statusFolder=$(find "$PIRExtensionFolderPath" -type d -name 'status')
+echo "statusFolder=$statusFolder"
+statusFile=$(ls $statusFolder/*.status)
+guestProxyAgentExtensionStatusObjGenerated=false
+guestProxyAgentExtensionServiceStatus=false
+timeout=300
+elpased=0
+while :; do 
+    extensionStatus=$(cat "$statusFile" | jq -r '.[0].status.status')
+    if [[ "$extensionStatus" == "success" ]]; then
+        guestProxyAgentExtensionStatusObjGenerated=true
+        guestProxyAgentExtensionServiceStatus=true
+        echo "The status is success."
+        break
+    fi
+    ((elapsed += interval))
+    if [[ $elapsed -ge $timeout ]]; then
+        echo "Timeout reached. Exiting the loop."
+        break
+    fi
+    sleep 5
+done
+
+echo "TEST: Check that process ProxyAgentExt is running"
+processId=$(pgrep ProxyAgentExt)
+echo "processId=$processId"
+if [ -z "$processId" ]; then
+    echo "Process ProxyAgentExt is not running"
+    guestProxyAgentExtensionServiceExist=false
+    guestProxyAgentExtensionProcessExist=false
+else 
+    echo "Process ProxyAgentExt is running"
+    guestProxyAgentExtensionServiceExist=true
+    guestProxyAgentExtensionProcessExist=true
+fi
+
+echo "TEST: Check that detailed status of the extension status to see if the Instance View is successful"
+proxyAgentstatus=$(cat "$statusFile" | jq -r '.[0].status.substatus[1].formattedMessage.message')
+guestProxyAgentExtensionInstanceView=false
+if [[ $proxyAgentstatus == *"SUCCESS"* ]]; then
+    echo "Instance View is successful" 
+    guestProxyAgentExtensionInstanceView=true
+else
+    echo "Instance View is not successful"
+fi
+
```

Comment:
if running validation after install, check extension version is 9.9.9 instead of 1.0.0 (pir version)